big-red-shuttle/Location.swift

```diff
@@ -45,6 +50,9 @@ class Location: NSObject, CLLocationManagerDelegate {
             print("longitude: \(location.longitude), latitude: \(location.latitude)")
             currentUserLocation = Coordinate(location: location, timestamp: Date())
         }
+        
+        fetchedUserLocationCompletionBlock?()
+        fetchedUserLocationCompletionBlock = nil
```

Comment:
This seems like a really bizarre API choice. Is the `fetchedUserLocationCompletionBlock` callback single use? It's not clear where the client would set this property considering it is set to nil as soon as it is used by delegate call.